### PR TITLE
Ins 3436 contract beautify

### PR DESCRIPTION
**Pull request by LinnikD**
Removed many unused fields and params. Fix logic errors

Note: no changes in functests, except deleted hellowrold

### CODE REVIEW

File: application/genesis/genesis.go [hunk 1]
```diff
@@ -286,11 +286,11 @@ func (g *Genesis) storeContracts(ctx context.Context) error {
 
 		states = append(states, contracts.GetDepositGenesisContractState(
 			application.DefaultDistributionAmount,
-			int64(pulse.OfUnixTime(ApplicationIncentivesLockup)),
+			int64(pulse.OfUnixTime(ApplicationIncentivesUnholdDate-pulse.MinTimePulse)),
```
Comment (ita-sammann) [inline application/genesis/genesis.go]: You subtract pulse number from timestamp

File: application/genesis/genesis.go [hunk 2]
```diff
@@ -314,11 +314,11 @@ func (g *Genesis) storeContracts(ctx context.Context) error {
 		states = append(states, contracts.GetAccountGenesisContractState("0", application.GenesisNameNetworkIncentivesAccounts[i], application.GenesisNameRootDomain))
 		states = append(states, contracts.GetDepositGenesisContractState(
 			application.DefaultDistributionAmount,
-			int64(pulse.OfUnixTime(NetworkIncentivesLockup)),
+			int64(pulse.OfUnixTime(NetworkIncentivesUnholdDate-pulse.MinTimePulse)),
```
Comment (ita-sammann) [inline application/genesis/genesis.go]: You subtract pulse number from timestamp

File: application/genesis/genesis.go [hunk 3]
```diff
@@ -342,11 +342,11 @@ func (g *Genesis) storeContracts(ctx context.Context) error {
 		states = append(states, contracts.GetAccountGenesisContractState("0", application.GenesisNameFoundationAccounts[i], application.GenesisNameRootDomain))
 		states = append(states, contracts.GetDepositGenesisContractState(
 			application.DefaultDistributionAmount,
-			int64(pulse.OfUnixTime(FoundationLockup)),
+			int64(pulse.OfUnixTime(FoundationUnholdDate-pulse.MinTimePulse)),
```
Comment (ita-sammann) [inline application/genesis/genesis.go]: You subtract pulse number from timestamp

File: application/genesis/genesis.go [hunk 4]
```diff
@@ -370,11 +370,11 @@ func (g *Genesis) storeContracts(ctx context.Context) error {
 		states = append(states, contracts.GetAccountGenesisContractState("0", application.GenesisNameFundsAccounts[i], application.GenesisNameRootDomain))
 		states = append(states, contracts.GetDepositGenesisContractState(
 			application.DefaultDistributionAmount,
-			FundsLockup,
+			int64(pulse.OfUnixTime(FundsUnholdDate-pulse.MinTimePulse)),
```
Comment (ita-sammann) [inline application/genesis/genesis.go]: You subtract pulse number from timestamp

File: application/genesis/genesis.go [hunk 5]
```diff
@@ -398,11 +398,11 @@ func (g *Genesis) storeContracts(ctx context.Context) error {
 		states = append(states, contracts.GetAccountGenesisContractState("0", application.GenesisNameEnterpriseAccounts[i], application.GenesisNameRootDomain))
 		states = append(states, contracts.GetDepositGenesisContractState(
 			application.DefaultDistributionAmount,
-			EnterpriseLockup,
+			int64(pulse.OfUnixTime(EnterpriseUnholdDate-pulse.MinTimePulse)),
```
Comment (ita-sammann) [inline application/genesis/genesis.go]: You subtract pulse number from timestamp

File: application/genesis/genesis.go (hunk 1 above)
Comment (SergeyRadist) [inline application/genesis/genesis.go]: I think you meant (pulse.OfUnixTime(ApplicationIncentivesUnholdDate)-pulse.MinTimePulse)

File: application/genesis/genesis.go (hunk 2 above)
Comment (SergeyRadist) [inline application/genesis/genesis.go]: same and further

**Pull request merged**